Wikipedia/Code/ArticleWebMessagingController.swift

```diff
@@ -40,7 +40,7 @@ class ArticleWebMessagingController: NSObject {
     }
     
     func addFooter(articleURL: URL, restAPIBaseURL: URL, menuItems: [PageContentService.Footer.Menu.Item], lastModified: Date?) {
-        guard let title = articleURL.wmf_title else {
+        guard let title = articleURL.percentEncodedPageTitleForPathComponents else {
```

Comment:
@joewalsh separate bug but may be related enough to fix in this PR - I notice footers are broken on articles with a '/' in the title. I see it in develop and this branch. I can make a separate task if that's preferable.

Reply:
@tonisevener 🤔I saw that as well and thought I verified that this change fixed it, but I'll double check. Which article are you using for testing?

Reply:
Looks like it's working for me on this branch with enwiki articles /dev/null, /Drive, and W:/2016Album/

Reply:
@joewalsh I'm seeing a missing footer on G/O Media.